sdk/web/armweb/ci.yml

```diff
@@ -0,0 +1,15 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  paths:
+    include:
+    - sdk/web/armweb/
+
+pr:
+  paths:
+    include:
+    - sdk/web/armweb/
+
+stages:
+- template: ../../../eng/pipelines/templates/jobs/archetype-sdk-client-samples.yml
+  parameters:
+    ServiceDirectory: 'web/armweb'
```

Comment:
@benbp not sure of the current state of the go common templates but I know in the other languages there are some assumptions that ServiceDirectory is only a single name not a full relative path. This pattern might cause some issues with common infrastructure. If this really isn't intended to be a ServiceDirectory but just a path for another template we might want to give it a different name to avoid any potential future confusion.